packages/tofu/src/inspectPrimordialAssignments.js

```diff
@@ -47,16 +69,30 @@ function inspectPrimordialAssignments(
   return results
 }
 
+/**
+ *
+ * @param {MemberLikeExpression} node
+ * @returns {string[]}
+ */
 function memberExpressionChainToPath(node) {
+  /** @type {string[]} */
   const path = []
   // walk down property chain
   while (isMemberLikeExpression(node.object)) {
-    path.push(node.property.name)
-    node = node.object
+    if ('name' in node.property) {
+      path.push(node.property.name)
+      node = node.object
+    } else {
+      break
+    }
```

Comment:
This may be a logic change. `node.object` is assigned to `node` 
Are we sure it's not possible for a name being absent from property (or property not being there) while the object field exists?

Reply:
I'll take another look at this and explain myself

Reply:
I've yanked this out and will put the logic changes in another PR